packages/discovery-react-components/src/components/CIDocument/CIDocument.tsx

```diff
@@ -1,20 +1,20 @@
 import CIDocument, {
-  CIDocumentProps,
+  CIDocumentProps as _CIDocumentProps,
```

Comment:
why is this change necessary

Reply:
Because this file contains both an instance and a type named `CIDocumentProps`. It looks like this was "allowed" by previous versions of TypeScript, but now it complains.